pkg/client/monitoring/v1/openapi_generated.go

```diff
@@ -1217,7 +1217,7 @@ func schema_pkg_client_monitoring_v1_PrometheusSpec(ref common.ReferenceCallback
 					},
 					"retention": {
 						SchemaProps: spec.SchemaProps{
-							Description: "Time duration Prometheus shall retain data for. Default is '24h'.",
+							Description: "Time duration Prometheus shall retain data for. Default is '24h', and must match the regular expression `[0-9]+(mssmhdwy)` (milliseconds seconds minutes hours days weeks years).",
```

Comment:
Sorry for holding back your PR with this unrelated issue, but shouldn't the RegEx be `[0-9]+(ms|s|m|h|d|w|y)`?

Reply:
Yup, good catch. `mssmhdwy` does not make sense in the regex context. I simply copied the sentences and expected them to be good after getting through review already 😇